content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -116,23 +116,23 @@ Queued experiment '4109ead' for future execution.
 
 ### How are experiments queued?
 
-Queued experiments are created similar to
-[Git stash](https://www.git-scm.com/docs/git-stash). The last experiment queued
-is found in `.git/refs/exps`, and earlier ones are in its [reflog].
+Queued experiments are managed using [dvc-task] and [Celery].
 
-[reflog]:
-  https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefreflogareflog
+[dvc-task]: https://github.com/iterative/dvc-task
+[celery]: https://docs.celeryq.dev/en/stable/index.html
 
 </details>
 
-Run them all with the `--run-all` flag:
+Run them all with `dvc queue start`:
 
 ```dvc
-$ dvc exp run --run-all
+$ dvc queue start
```

Comment:
Thanks for the guide update @pmrowla 

Is `exp run --run-all` still available or does it get hard-deprecated after this release?

Because we have several other guides and Get Started pages that mention it (on me, need to prioritize depending on your answer).